ocp_resources/node_network_configuration_policy.py

```diff
@@ -370,32 +370,26 @@ def _process_failed_status(self, failed_condition_reason):
 
         raise NNCPConfigurationFailed(f"Reason: {failed_condition_reason}\n{last_err_msg}")
 
+    @retry(wait_timeout=120, sleep=5)
     def wait_for_status_success(self):
         failed_condition_reason = self.Conditions.Reason.FAILED_TO_CONFIGURE
         no_match_node_condition_reason = self.Conditions.Reason.NO_MATCHING_NODE
 
-        # if we get here too fast there are no conditions, we need to wait.
-        self.wait_for_configuration_conditions_unknown_or_progressing()
```

Comment:
This is not needed anymore?

Reply:
Does not look like. I ran the tests many times.

Reply:
then I think you can remove the function as well.

Reply:
I checked. It is currently used in https://github.com/RedHatQE/openshift-virtualization-tests/blob/main/tests/network/nmstate/test_nmstate_sanity.py#L211

Reply:
ok